frontend/app/templates/components/transportation/existing-conditions/existing-conditions-steps.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="ui pointing menu">
```

Comment:
should this be above the tbale instead of above the map, like in the public schools view?